### PR TITLE
fix(core): handle text overflow styles for search box

### PR DESCRIPTION
**Pull request by SOUMITRO-SAHA**
### Description:

This PR implements dynamic text handling for the search box to ensure that text is truncated with an ellipsis when it exceeds the box's size. This enhancement improves the display of search box labels across different languages by ensuring that text is consistently presented, even when it is longer than the available space.

### Changes Made:

- Added CSS to dynamically apply the ellipsis property to the search box text.
- Ensured that the truncation behaviour adjusts based on the size of the search box, making it suitable for various languages.

### Benefits:

- Provides a more consistent and readable user interface.
- Enhances support for multi-language text, preventing overflow and maintaining a clean design.

image

image

### CODE REVIEW

File: packages/frontend/core/src/components/app-sidebar/quick-search-input/index.css.ts [hunk 1]
```diff
@@ -28,3 +28,9 @@ export const shortcutHint = style({
   color: cssVarV2('text/tertiary'),
   fontSize: cssVar('fontBase'),
 });
+export const quickSearchBarEllipsisStyle = style({
+  maxWidth: '200px',
```
Comment (JimmFly) [inline packages/frontend/core/src/components/app-sidebar/quick-search-input/index.css.ts]:
```suggestion
```
Remove this

Comment (JimmFly) [reply packages/frontend/core/src/components/app-sidebar/quick-search-input/index.css.ts]: and add `overflow: 'hidden',` to `root` class
```
export const root = style({
...
  overflow: 'hidden',
});
```

Comment (SOUMITRO-SAHA) [reply packages/frontend/core/src/components/app-sidebar/quick-search-input/index.css.ts:32]: You’re right about the maxWidth, but after setting `overflow: 'hidden'` on the root, it now looks like this. Is it okay?

image

Comment (SOUMITRO-SAHA) [reply packages/frontend/core/src/components/app-sidebar/quick-search-input/index.css.ts]: You were absolutely right about the maxWidth. I realise now that it wasn’t necessary at all. I’m not sure what I was thinking at the time. Great code review, much appreciated @JimmFly

**Review by JimmFly — APPROVED** [review]

**Pull request merged**